test/strategies/explicit.jl

```diff
@@ -0,0 +1,54 @@
+good = KNNClassifier(K=2)
+bad = KNNClassifier(K=10)
+ugly = ConstantClassifier()
+
+r = [good, bad, ugly]
+
+rng=StableRNGs.StableRNG(123)
+X, y = make_blobs(rng=rng)
+
+@testset "constructor" begin
+true 
```

Comment:
TODO, I guess

Reply:
Oops. Forgot about that. It was the `TunedModel` constructor that needed new/altered tests and these are at test/tuned_model.jl instead. I don't think `Explicit()` which has no fields needs a separate test 😄